ROOTFS/etc/waggle/sanity/fatal/static_psu_control.test

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+
+# Written by Kenan with plenty of Joe's code mixed in
+
+# A static test to make sure all of the PSU stuff works
+
+# the other interactive tests have this 'Begin' line so I'll also include it
+
+pf="Static PSU test"
+echo "$pf: Begin"
+
+acok=$(waggle-get-config -s wagman -k gpio-acok3)
+pg1=$(waggle-get-config -s wagman -k gpio-pg1)
+pg2=$(waggle-get-config -s wagman -k gpio-pg2)
+pg3=$(waggle-get-config -s wagman -k gpio-pg3)
+pg4=$(waggle-get-config -s wagman -k gpio-pg4)
+
+gpiopath="/sys/class/gpio"
+
+
+# not my code, copied from Joe's PSU test
+gpio_high() { 
+
+        gpio=$1
+        if [ ! -f $gpiopath/gpio$gpio/value ]; then 
+                echo "$pf: Cannot find value of device $gpio"
+                exit 1
+        fi
+
+        if [ $(cat $gpiopath/gpio$gpio/value) -eq 1 ]; then 
+                return 0
+        else 
+                return 1
+        fi
+}
+
+# Not my code, taken from Joe's PSU test
+gpio_cleanup()
+{
+        if [ -d $gpiopath/gpio$acok ]; then
+                echo $acok > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg1 ]; then
+                echo $pg1 > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg2 ]; then
+                echo $pg2 > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg3 ]; then
+                echo $pg3 > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg4 ]; then
+                echo $pg4 > $gpiopath/unexport
+        fi
+}
+trap gpio_cleanup EXIT
```

Comment:
So, i wouldn't do a trap cleanup in your code.  Your code is going to run once an hour and it is okay for the gpios to remain exported.  In fact, you don't want to "unexport" something that some other service may be using.  Imagine the scenario where some other service is attempting to read the GPIO valus at the same time your code runs.  Your code would "unexport" the GPIOs in the middle of that other service running.

Yes, it is nice to clean-up after ourselves and return the system to the state it was before we ran.  but in this case an exported GPIO is not harmful.

The reason I did it in the "interactive" test is that is run manually, by hand, once.  It is technically not necessary in my use-case either. 

I would prefer to not see this auto "unexporting" as it can cause more harm then good.